change/@fluentui-react-components-1b3e39f8-2025-47cb-bf2e-c541853c97b8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
This does not affect published code, so should be `none`. @khmakoto can you please create a followup to update change files?

Reply:
Will do!